includes/class-wcpdf-settings-debug.php

```diff
@@ -348,6 +373,66 @@ public function import_settings( $data ) {
 		}
 	}
 	
+	public function reset_settings( $data ) {
+		extract( $data );
+		
+		if ( empty( $type ) ) {
+			$message = __( 'Reset settings type is empty!', 'woocommerce-pdf-invoices-packing-slips' );
+			wcpdf_log_error( $message );
+			wp_send_json_error( compact( 'message' ) );
+		}
+		
+		$settings_option = '';
+		
+		switch ( $type ) {
+			case 'general':
+				$settings_option = 'wpo_wcpdf_settings_general';
+				break;
+			case 'debug':
+				$settings_option = 'wpo_wcpdf_settings_debug';
+				break;
+			default:
+				$settings_option = apply_filters( 'wpo_wcpdf_reset_settings_option', $settings_option, $type );
```

Comment:
I think it will be better to make this hook, and the hook used in the `export_settings` method line 272 have the same pattern in their name.

Reply:
Can you explain better?

Reply:
Sure, I meant it might be better to write it as `wpo_wcpdf_reset_settings`, the same as `wpo_wcpdf_export_settings` without the `option` at the end or adding an `_option` at the end of the `wpo_wcpdf_export_settings`.

Reply:
Are different things, in the `wpo_wcpdf_export_settings` the hook return value is the settings itself, and `array`, and in the `wpo_wcpdf_reset_settings_option` is the settings option key, a `string`.